frontend/src/styles/globals.scss

```diff
@@ -28,6 +28,11 @@ body,
 #__next,
 #overlayProvider {
   font-family: $font-stack-base;
+
+  @media screen and #{$mq-lg} {
+    // Ensures that the footer stays at the bottom
+    height: 100vh;
```

Comment:
Question: What does this do when there's more content than `100vh`?  Should this be `min-height` instead? Does that have the same effect? 

Also, would doing a [screen _height_ media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/height) be helpful?

Reply:
Yeah, IIRC the original was `100%`, which should allow it to grow to the full height of its parent. (Specifically, the footer is in an element with `display: flex; flex-direction: vertical`, and the element above the footer has a `flex-grow` higher than 0. That allows it to push the footer to the bottom, as long as all its parents elements have `height: 100%`.)

Which is not to say that `100vh` might not work or that you may not have good reasons for using that - just sharing the context behind the original value :)